src/pybind/mgr/dashboard/tools.py

```diff
@@ -49,8 +49,7 @@ def request_begin(self):
         req = cherrypy.request
         user = JwtManager.get_username()
         # Log the request.
-        logger.debug('[%s:%s] [%s] [%s] %s', req.remote.ip, req.remote.port,
-                     req.method, user, req.path_info)
+        self._request_log(logger.debug)
```

Comment:
Logging the request is different from logging the response (the `_request_log` method name is misleading as it is actually logging the response of the "request"). In the request there is no "status", "latency", or content length. You should revert this change.

Reply:
We are printing 2 lines that repeat the same 4 fields (req.remote.ip, req.remote.port, req.method, req.path_info) and only differ in 3.

Most webservers  ([apache](https://httpd.apache.org/docs/2.4/logs.html#page-header) and [nginx](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/)) only log one line per request-response pair (including request data + "status" or optionally other response data). This is an Apache access log line ([format](https://httpd.apache.org/docs/2.4/mod/mod_log_config.html#formats)):

`127.0.0.1 - peter [9/Feb/2017:10:34:12 -0700] "GET /sample-image.png HTTP/2" 200 1479`

I think we should do that and also use a separate file for event/access log with the [W3C log format](https://www.w3.org/TR/WD-logfile.html). I'll revert this and create a ticket with this proposal.